src/app/campaign.service.ts

```diff
@@ -182,10 +182,10 @@ export class CampaignService {
   }
 
   getCampaignImpactStats(): Observable<CampaignStats>{
-    // temporarily hardcoded until the endpoint is ready on SF side
-    // TODO: replace with the commented line below
-    return of(new CampaignStats(242_309_804, 15_000));
-    // return this.http.get<CampaignStats>(`${environment.apiUriPrefix}${this.apiPath}/campaigns/stats`)
```

Comment:
I think before we merge this we should switch to using a route resolver in the component instead of using the observable directly - see e.g. https://www.digitalocean.com/community/tutorials/angular-route-resolvers

Reply:
Also we probably need to fix the staging version of the API in SF + maybe think about what the frontend should do in case of an error in salesforce.